sample-code/src/orchestration.ts

```diff
@@ -7,10 +7,10 @@ import {
  * Ask GPT about the capital of France.
  * @returns The answer from the orchestration service in Gen AI Hub.
  */
-export function orchestrationCompletion(): Promise<CompletionPostResponse> {
+export async function orchestrationCompletion(): Promise<CompletionPostResponse> {
   const orchestrationClient = new OrchestrationClient();
 
-  return orchestrationClient.chatCompletion(
+  const response = await orchestrationClient.chatCompletion(
```

Comment:
It's a little bit of a red flag the same type can be used even though you now only return the data property.

Should the type be different / more narrow, or do you think it's fine?